cli/tests/integration/repl_tests.rs

```diff
@@ -912,11 +912,11 @@ fn repl_unit_tests() {
     console.expect("  step1 ... ok (");
     console.expect("test1 ... ok (");
     console.expect("test2 ... FAILED (");
-    console.expect(" ERRORS ");
+    console.expect("ERRORS");
```

Comment:
These repl tests weren't working on windows because windows modifies the output a bit. I updated the tests here to be more reliable.